src/routes/v1/live-status.ts

```diff
@@ -0,0 +1,106 @@
+import { Request, Router } from 'express'
+
+import { Locale } from '../../utils/locale'
+import { between } from '../../utils/random'
+import { equalStatus, getLiveStatus, Status } from '../../logic/otp2/liveStatus'
+import { addMinutes, isAfter } from 'date-fns'
+
+const router = Router()
+
+const STREAM_TIMEOUT = 30 // minutes
+
+type EventType = 'live-status' | 'close' | 'timeout'
+
+function createEvent<T>(id: number, type: EventType, data?: T): string {
+    const serializedData = data ? JSON.stringify(data) : ''
+
+    return `event:${type}\nid:${id}\ndata:${serializedData}\n\n`
+}
+
+router.get(
+    '/:serviceJourneyId',
+    async (
+        req: Request<
+            { serviceJourneyId: string },
+            Record<string, never>,
+            Record<string, never>,
+            { date: string; locale?: Locale }
+        >,
+        res,
+        next,
+    ) => {
+        try {
+            const serviceJourneyId = req.params.serviceJourneyId
+            const { date, locale } = req.query
+
+            res.set({
+                'Cache-Control': 'no-cache',
+                'Content-Type': 'text/event-stream',
+                Connection: 'keep-alive',
+            })
+            res.flushHeaders()
+
+            let eventId = 0
+            let interval: NodeJS.Timer | undefined = undefined
```

Comment:
hvordan fungerer den typingen (hvis det er det) med 'undefined = undefined' hvorfor ikke bare undefined?

Reply:
`= undefined` biten er for å initialisere variabelen :) 

Jeg trodde at den automatisk vill bli satt til undefined men kanskje ts klager om du ikke gjør det eksplisitt? 🤔